pkg/maps/ctmap/ctmap.go

```diff
@@ -294,10 +294,7 @@ func newMap(mapName string, m mapType) *Map {
 }
 
 func purgeCtEntry6(m *Map, key CtKey, natMap NatMap) error {
-	err := m.Delete(key)
-	if err == nil && natMap != nil {
-		natMap.DeleteMapping(key.GetTupleKey())
-	}
+	err := natMap.DeleteMapping(key.GetTupleKey())
```

Comment:
Why did you remove the CT entry removal?

Reply:
Sorry the comment was not clear. I should have asked for clarity.